youtube_dl/extractor/liveleak.py

```diff
@@ -134,11 +138,12 @@ def _real_extract(self, url):
                 orig_url = re.sub(r'\.mp4\.[^.]+', '', a_format['url'])
                 if a_format['url'] != orig_url:
                     format_id = a_format.get('format_id')
-                    formats.append({
-                        'format_id': 'original' + ('-' + format_id if format_id else ''),
-                        'url': orig_url,
-                        'preference': 1,
-                    })
+                    if self._is_valid_url(orig_url, video_id, format_id):
```

Comment:
the message will be `Checking None URL` if `format_id` is not available, provide a fallback for `format_id`.

Reply:
Is 9ac2735 ok?